config/buildspec_build_wheel.yml

```diff
@@ -14,7 +14,7 @@ phases:
         - apt-get install sudo -qq -o=Dpkg::Use-Pty=0
         - sudo apt-get update -qq -o=Dpkg::Use-Pty=0
         - sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0
-        - cd $CODEBUILD_SRC_DIR && chmod +x config/protoc_downloader.sh && ./config/protoc_downloader.sh
+        - cd $CODEBUILD_SRC_DIR
```

Comment:
Nothing else in the install stage requires this directory, can we remove this line entirely?

Same for all of the other buildspecs too.